en-US/firefox/features/bookmarks.lang

```diff
@@ -18,53 +20,30 @@ Better bookmarks
 Don’t agonize, lovers of bookmarking. Organize with Firefox.
 
 
-;Features
-Features
-
-
+# File means to arrange (that bookmark).
 ;File that bookmark
 File that bookmark
 
 
+# Dial in means to access (the bookmarks).
 ;Get your faves all sorted with the bookmark star icon, which lets you add custom names and folders quickly. Then dial in your bookmarks toolbar to make sure you never lose sight of the links you love.
 Get your faves all sorted with the bookmark star icon, which lets you add custom names and folders quickly. Then dial in your bookmarks toolbar to make sure you never lose sight of the links you love.
 
 
+# Fly here means to have your bookmark go wherever you go.
 ;Fly with that bookmark
 Fly with that bookmark
 
 
+# On the fly means doing something quickly without giving it too much thought.
 ;Take your favorites on the fly. Use Firefox Sync to access your bookmarks across all your devices, from desktop to mobile. Or try <a href="%(url)s">Pocket</a> to save any online article, or page and come back to later – even without internet.
 Take your favorites on the fly. Use Firefox Sync to access your bookmarks across all your devices, from desktop to mobile. Or try <a href="%(url)s">Pocket</a> to save any online article, or page and come back to later – even without internet.
 
 
+# Futz with means to busy with oneself.
```

Comment:
The word is an old expression. It means to busy with oneself aimlessly and probably not something meaningful. We need to avoid using those words in this context, :).